Update README.md (#132)
terraform-ibm-modules/terraform-ibm-account-infrastructure-base: solutions/account-infrastructure-base/README.md

# IBM Cloud Account Infrastructure Base solution

> This is an end-to-end deployable architecture solution that provisions the following:

Reviewer: ```suggestion
An end-to-end deployable architecture solution that provisions the following infrastructure:
```

- A resource group
- A default set of Financial Services Cloud compliant account settings
- An IBM Cloud Object Storage instance
- An IBM Cloud Object Storage bucket encrypted with the key management service key
- An Activity Tracker target for the Object Storage bucket
- An Activity Tracker route to route events to the Object Storage bucket

> - A trusted profile to give Projects access to securely deploy this solution

Reviewer 2: The trusted profile for projects is not for this solution, but for deploying other solutions in this account via that trusted profile so maybe something like `- A trusted profile to give Projects access to securely deploy solutions in this account`

Author: That sounds good, thank you! @MatthewLemmond

> - A resource group
> - A default set of Financial Services Cloud compliant account settings
> - An IBM Cloud Object Storage instance
> - An IBM Cloud Object Storage bucket encrypted with the key management service key
> - An Activity Tracker target for the Object Storage bucket
> - An Activity Tracker route to route events to the Object Storage bucket
> - A trusted profile to give Projects access to securely deploy solutions in this account

Reviewer: ```suggestion
- A resource group
- A default set of account settings that are compliant with the IBM Cloud for Financial Services framework 
- An IBM Cloud Object Storage instance
- An IBM Cloud Object Storage bucket encrypted with the key management service key
- An Activity Tracker target for the Object Storage bucket
- An Activity Tracker route that routes events to the Object Storage bucket
- A trusted profile to give access for IBM Cloud Projects to deploy solutions securely in this account
```

![account-infrastructure-base](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-account-infrastructure-base/main/reference-architectures/base-account-enterprise.svg)

## Before you begin

> Create an IAM authorization policy in the account where the Key Management Service (KMS) key resides. Grant the Object Storage service in the account where this solution is deployed the Reader role on the KMS instance that the key belongs to.

Reviewer: ```suggestion
Create an IAM authorization policy in the account where the key management service (KMS) key resides. Grant the Object Storage service the Reader role on the KMS instance that the key belongs to in the account where this solution is deployed.
```

## Limitations

> The solution currently does not support configuring the following settings that are required for FSCloud compliance:
> - The user list visibility IAM setting. An account owener can enable this restriction manually in the IBM Cloud console by following these [steps](https://cloud.ibm.com/docs/account?topic=account-iam-user-setting).
> - The Financial Services Validated setting. An account owner can enable the account to be Financial Services Validated designated, which means your account stores and manages regulated financial services information. Follow these [steps](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated) to set this manually in the IBM Cloud console.

Reviewer: ```suggestion

The solution does not support configuring the following settings that are required for compliance with IBM Cloud for Financial Services:

- The user list visibility IAM setting. An account owner can enable this restriction in the IBM Cloud console by following these [steps](https://cloud.ibm.com/docs/account?topic=account-iam-user-setting).
- The Financial Services Validated setting. An account owner can enable the account to be designated as Financial Services Validated. IBM Cloud for Financial Services Validated designates that an IBM Cloud service or ecosystem partner service has evidenced compliance to the controls of the IBM Cloud Framework for Financial Services and can be used to build solutions that might themselves be validated. 

    For more information, see [Enabling your account to use Financial Services Validated products](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated).
```
